www/js/util.js

```diff
@@ -3112,3 +3060,95 @@ function stopQueueSpinner(data) {
         $("#queueprogress").remove();
     }
 }
+
+function CSEmoteList(selector) {
+    EmoteList.call(this, selector);
+}
+
+CSEmoteList.prototype = Object.create(EmoteList.prototype);
+
+CSEmoteList.prototype.loadPage = function (page) {
+    var tbody = this.table.children[1];
+    tbody.innerHTML = "";
+
+    var start = page * this.itemsPerPage;
+    if (start >= this.emotes.length) {
+        return;
+    }
+    var end = Math.min(start + this.itemsPerPage, this.emotes.length);
+    var self = this;
+
+    for (var i = start; i < end; i++) {
+        var row = document.createElement("tr");
+        tbody.appendChild(row);
+
+        (function (emote) {
+            // Add delete button
+            var tdDelete = document.createElement("td");
+            var btnDelete = document.createElement("button");
```

Comment:
`btnDelete` has no click handler to actually delete an emote. :smirk: I do think when you add that, Henke's suggestion was good: Populate the new emote form with the values of the deleted emote in case of accidental click, since you can't stack Bootstrap modals without hacks or additional libraries to make a confirmation dialog.